expose addImportMap (#2429)
systemjs/systemjs: docs/api.md

  console.log(ns); // { exportName: 'value' }
};
```

#### System.addImportMap(map [, base])
Type: `Function`

> Allows you to add import-map without DOM interop.

Reviewer: ```suggestion
Allows adding an import map without using the DOM.
```
